### PR TITLE
revise def and synonym for gastric antral vascular ectasia

### LINKED ISSUE #7345: Watermelon stomach
Watermelon stomach

Gastric antral vascular ectasia (GAVE)

Gastric antral vascular ectasia (GAVE) is a [rare disorder that causes severe acute gastrointestinal bleeding. Also known as watermelon stomach, due to its endoscopic appearance,  this condition is marked by dilated capillaries localized to the antrum that give the appearance of watermelon streaks. The dilated vessels render the affected areas more susceptible to bleeding. Even though the cause is still unknown. The number of cases in the general population is not known but is rare.  It occurs in about 4% of non-variceal upper GI bleedings and 6% of upper GI bleeds in patients with liver cirrhosis. The exact cause is unknown; however, it is often associated with several conditions such as cirrhosis and autoimmune diseases,  systemic sclerosis, and CREST syndrome. GAVE has become a distinct entity with established co-morbid conditions. Treatment can be done with endoscopic coagulation of the lesions. 

https://www.sciencedirect.com/science/article/pii/S2210261222007209
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8110233/
https://www.sages.org/wiki/gastric-antral-vascular-ectasia/

Parent term: Rare hemorrhagic disorder

Thank you, Gioconda

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #7345

### CODE REVIEW

File: src/ontology/mondo-edit.obo
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:440187]: for some reason this change is coming through on all my PRs today. I see it on master. It probably won't resolve until my PRs are merged but it can be ignored.

**Comment by twhetzel** [comment]
@nicolevasilevsky should this wait to be closed until we hear back on the ticket https://github.com/monarch-initiative/mondo/issues/7345?

**Comment by nicolevasilevsky** [comment]
No, I think we can merge this PR without waiting for Gioconda's response.

**Comment by nicolevasilevsky** [comment]
I can do a separate PR if further action is needed

**Comment by nicolevasilevsky** [comment]
yes, the rare tag needs to be added, but it can be done on a separate PR so this one does not sit. Unless you can add it on this PR?

**Comment by twhetzel** [comment]
ok, i am adding the rare tag now.

**Review by twhetzel — APPROVED** [review]
lgtm

**Pull request merged**
